scripts/addons-namespaces/generate-add-ons-list.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import os
+import string
+import argparse
+import oyaml as yaml
+
+'''
+This is a script for finding the manespace for each addon operator in managed-tenants repo and outputting them in a yaml file.  
+'''
+
+def main():
+
+    parser = argparse.ArgumentParser(
+        description="Finds all namespaces for managed-tenants addons and outputs them as a yaml document"
+    )
+    parser.add_argument(
+        "--path",
+        "-p",
+        required=True,
+        help="The path to managed-tenants/addons/ directory [required]",
+    )
+    parser.add_argument(
+        "--output",
+        "-o",
+        help="The destination of the output yaml file. Default managed-cluster-config/resources/addons-namespaces/main.yaml ",
```

Comment:
No need to indicate the default in the help message. Just add a default value here.